cmd/selectionMCMC/selectionMCMC.go

```diff
@@ -8,16 +8,20 @@ import (
 	"log"
 )
 
-func selectionMCMC(filename string, outFile string, muZero float64, sigmaZero float64, iterations int, randSeed bool, setSeed int64) {
+func selectionMCMC(filename string, outFile string, muZero float64, sigmaZero float64, iterations int, randSeed bool, setSeed int64, unPolarized bool) {
 	common.RngSeed(randSeed, setSeed)
-	data := popgen.VcfToAFS(filename)
-	popgen.MetropolisHastings(data, muZero, sigmaZero, iterations, outFile)
+	data, err := popgen.VcfToAFS(filename, !unPolarized) //VcfToAFS is writted with polarized as the argument for clarity, so the bool is flipped here.
+	if err != nil {
```

Comment:
I think there is an error handling package now that will shorten this to one line.